pygama/dsp/transforms.py

```diff
@@ -867,76 +973,3 @@ def peakdet_test(waves, calcs, delta, sigma, ihi, test=False):
     # ok, i hereby bless the "partially vectorized" peakdet function.
 
     exit()
-
-
-def cfd(waves, calcs, frac=0.5, delay=0.5, win=0.01, wfin="wf_blsub", test=False):
```

Comment:
Oops, you deleted some code I want to keep.  Can you please restore this block?  After that I will merge.  We can discuss the details of your processors by email -- here I just wanted to make sure they follow the structure of the other processors.